### PR TITLE
Wrap Graph with VG object before serialization

### PR DESCRIPTION
**Pull request by apregier**
This allows the messages to be split up in a
way that is valid for protobuf (not too large)

This fixed the error that I was seeing in a graph that I had constructed (https://github.com/vgteam/vg/issues/1647), and it doesn't seem _worse_ in terms of memory management than what was previously there - but I might be wrong about that.

### CODE REVIEW

**Comment by glennhickey** [comment]
Jenkins, OK to test.

**Comment by apregier** [comment]
I'm not sure what happened to the TravisCI tests?  When I looked yesterday it seemed that one had failed because of a timeout but the other has passed.  Now when I came back today they aren't linked anymore?

File: src/subcommand/construct_main.cpp [hunk 1]
```diff
@@ -198,12 +198,19 @@ int main_construct(int argc, char** argv) {
     auto callback = [&](Graph& big_chunk) {
         // TODO: these chunks may be too big to (de)serialize directly. For now,
         // just serialize them directly anyway.
+        bool sent = false;
+        function<bool(Graph&)> get_graph = [&](Graph& graph){
+            if (sent) {
+                return false;
+            }
+            graph = Graph();
```
Comment (adamnovak) [inline src/subcommand/construct_main.cpp]: Why construct an empty Graph here?

**Comment by adamnovak** [comment]
I think we ended up working around #1647 by just making the maximum chunk size very large with https://github.com/vgteam/vg/pull/1718, but this ought to fix it altogether.

**Comment by adamnovak** [comment]
I'm going to merge this, and do some stuff on top of it.

**Pull request merged**